lua/oil/adapters/files.lua

```diff
@@ -213,12 +213,28 @@ end
 M.normalize_url = function(url, callback)
   local scheme, path = util.parse_url(url)
   assert(path)
-  if fs.is_windows and path == "/" then
-    return callback(url)
+
+  if fs.is_windows then
+    if path == "/" then
+      return callback(url)
+    else
+      local drive = url:match("^oil:///(%u)$")
+      if drive then
+        return callback("oil:///" .. drive .. "/")
```

Comment:
This section handles navigating back to root where the drives are listed and allows navigating into them